src/Sylius/Bundle/ProductBundle/Resources/translations/validators.en.yml

```diff
@@ -40,13 +44,17 @@ sylius:
             max_length: Option name must not be longer than 1 character.|Option name must not be longer than {{ limit }} characters.
         values:
             min_count: Please add at least {{ limit }} option value.|Please add at least {{ limit }} option values.
+        locale:
+            not_blank: Please enter the locale.
```

Comment:
Maybe we can create some more generic translation key. We can create one translation key like this per bundle.